docs/components/Text/Text.tsx

```diff
@@ -14,6 +14,10 @@ interface Props<C extends React.ElementType> {
    * The style prop of the Text component
    */
   style?: object;
+  /**
+   * The color prop of the Text component
+   */
+  color?: string;
```

Comment:
Adding color prop and setting the default to text/default to fix bug with shadows with this inclusion of CSS variable stories 

### Before

https://github.com/MetaMask/design-tokens/assets/8112138/cdf9507b-d6a4-4218-8ebe-1ec4d53cbc7e

### After

https://github.com/MetaMask/design-tokens/assets/8112138/260317f0-1e26-4b46-8163-95dfb3847242